### PR TITLE
[15.0][MIG] account_invoice_force_number

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
This module allows to force the invoice numbering. It displays the move_name field.
If user fills that field, the typed value will be used as invoice (and move) number.
Otherwise, the next sequence number will be retrieved and saved.
So, the new field has to be used when user doesn't want to use the default invoice numbering for a specific invoice.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
please rebase

**Comment by JuanyDForgeflow** [comment]
> please rebase

Done!

**Comment by JuanyDForgeflow** [comment]
> please rebase

Done!

File: account_invoice_force_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2011-2016 Agile Business Group
+# Copyright 2017 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Force Invoice Number",
+    "version": "15.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline account_invoice_force_number/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: account_invoice_force_number/__manifest__.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply account_invoice_force_number/__manifest__.py]: @MiquelRForgeFlow done

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by andreampiovesana — APPROVED** [review]
Tested in production

**Comment by FSchiepers** [comment]
Hi,
I tested the module and I always get the following error:
The Date (01/19/2023) doesn't match the sequence number of the related Journal Entry (INV/2022-12-007)
You will need to clear the Journal Entry's Number to proceed.
In doing so, you might want to resequence your entries in order to maintain a continuous date-based sequence.

**Comment by AaronHForgeFlow** [comment]
Hi @FSchiepers , Can you reproduce the issue in runboat?

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Review by kaynnan — APPROVED** [review]

**Pull request merged**